pkgs/games/factorio/default.nix

```diff
@@ -14,19 +15,38 @@ let
 
   # NB If you nix-prefetch-url any of these, be sure to add a --name arg,
   #    where the ultimate "_" (before the version) is changed to a "-".
+  experimentalOrFinal = if experimental then "experimental" else "final";
```

Comment:
The variable could be named `branch` or something.

Also, the [website](https://www.factorio.com/) uses the following terms:

> **Latest releases**
> Stable: 0.15.40
> Experimental: 0.16.16

Reply:
I've made the changes you suggested.

Reply:
@jtojnar Is there any remaining issues?